Tests/NumLib/TestMeshComponentMap.cpp

```diff
@@ -27,7 +27,7 @@ class NumLibMeshComponentMapTest : public ::testing::Test
         using MeshComponentMap = NumLib::MeshComponentMap;
 
     public:
-        NumLibMeshComponentMapTest() : mesh(nullptr), cmap(nullptr)
+        NumLibMeshComponentMapTest()
```

Comment:
The mesh can be generated within the initializer list.

Reply:
I also put the mesh into a unique ptr.